modules/logging-content-filter-prune-records.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-content-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-content-filter-prune-records_{context}"]
+= Configuring content filters to prune log records
+
+When content filters are configured, the log collector edits log streams according to the filters before forwarding. If you configure the `prune` filter, the collector prunes log records by removing pod annotations or other low value fields.
```

Comment:
I might also say... prune log records by removing low value fields (e.g. pod annotation).  It sort of reads as though the intent is to explicitly remove annotations... instead of identifying that field as an example.

Reply:
are there other examples maybe you can give as well?

Reply:
`kubernetes.flat_labels` is one specifically though I wish I knew more about kube audit filters because you could prune those as well outside of @alanconway api filter framework